auth-web/src/components/auth/staff/continuation-application/HomeJurisdictionInformation.vue

```diff
@@ -231,122 +231,143 @@
-    return DateUtils.dateToPacificDate(date, true)
-  }
-
-  /**
-   * Whether a continuation in director affidavit is required.
-   * Is true if the business is a Continued In ULC from Alberta or Nova Scotia.
-   */
-  get isContinuationInAffidavitRequired (): boolean {
-    const entityType = this.continuationIn?.nameRequest?.legalType
-    const country = this.continuationIn?.foreignJurisdiction?.country
-    const region = this.continuationIn?.foreignJurisdiction?.region
-
-    return (
-      entityType === CorpTypes.ULC_CONTINUE_IN &&
-      country === 'CA' &&
-      (region === 'AB' || region === 'NS')
-    )
-  }
-
-  /** Downloads the director affidavit document. */
-  async downloadAffidavitDocument (): Promise<void> {
-    await this.download(this.continuationIn?.foreignJurisdiction?.affidavitFileKey,
-      this.continuationIn?.foreignJurisdiction?.affidavitFileName)
-  }
+    /** Downloads the specified document. */
+    async function download (documentKey: string, documentName: string): Promise<void> {
+      if (!documentKey || !documentName) return // safety check
+
+      state.isDownloading = true
+      await BusinessService.downloadDocument(documentKey, documentName).catch(error => {
+        // eslint-disable-next-line no-console
+        console.log('downloadDocument() error =', error)
+        state.dialogTitle = 'Unable to download document'
+        state.dialogText = 'An error occurred while downloading the document. Please try again.'
+        const v = errorDialogComponent.value as any; v.open()
+      })
+      state.isDownloading = false
+    }
 
-  /** Downloads the specified authorization document. */
-  async downloadAuthorizationDocument (item: { fileKey: string, fileName: string }): Promise<void> {
-    await this.download(item.fileKey, item.fileName)
-  }
+    /**
+     * Converts a date string to a Pacific date string.
+     * Sample input: "2024-07-01".
+     * Sample output: "July 1, 2024".
+     */
+    function strToPacificDate (str: string): string {
+      const date = moment(str).toDate()
+      return (date) ? moment(date).tz('America/Vancouver').format('MMMM D, YYYY') : ''
+    }
 
-  /** Downloads the specified document. */
-  private async download (documentKey: string, documentName: string): Promise<void> {
-    if (!documentKey || !documentName) return // safety check
-
-    this.isDownloading = true
-    await BusinessService.downloadDocument(documentKey, documentName).catch(error => {
-      // eslint-disable-next-line no-console
-      console.log('downloadDocument() error =', error)
-      this.dialogTitle = 'Unable to download document'
-      this.dialogText = 'An error occurred while downloading the document. Please try again.'
-      this.$refs.errorDialog.open()
-    })
-    this.isDownloading = false
+    return {
+      errorDialogComponent,
+      downloadAffidavitDocument,
+      downloadAuthorizationDocument,
+      ...toRefs(state)
```

Comment:
`...toRefs(state)` here, above `...state` should they be the same?

Reply:
It's my understanding that `toRefs()` needs to be used because I declare a ref on line 256.

I will try `...state` and see if it still works.

Reply:
all good, either way works for me

Reply:
Looks like state isn't correctly injected into the component if I don't use `toRefs()` -- the modal looks like this (missing its props):

![image](https://github.com/user-attachments/assets/10c6b58c-e1cf-4f3d-b012-7854b224fcb3)